Update the path to the import context directory (#3010)
neo4j/docs-operations: modules/ROOT/pages/changes-2025-2026.adoc

A new parameter `--debug` is added as an alias for the `--log` option.

== Logging

Neo4j 2026.04::

> The log that the `neo4j-admin database import` command creates to store information about the import process, is moved from _server/data/imports_ to _server/logs/_.

Author: ```suggestion
The directory that the `neo4j-admin database import` command creates to store information about the import process, is moved from _server/data/imports_ to _server/logs/_.
```

Neo4j 2026.03::

> The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_ and the path to it are updated to _server/data/imports/dbname-yyyy-MM-dd.HH.mm.ss/import.log_. +

Author: ```suggestion
The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_  has been changed to _dbname-yyyy-MM-dd.HH.mm.ss/import.log_, as the `neo4j-admin database import` command creates a new directory from version 2026.03. +
The path to the directory is changed from `server/logs` to `server/data/imports/`.

```

A new parameter `--debug` is added as an alias for the `--log` option.

== Logging

Neo4j 2026.04::

> The directory that the `neo4j-admin database import` command creates to store information about the import process, is changed back to _server/logs/_.

Author: ```suggestion
The directory that the `neo4j-admin database import` command creates to store information about the import process is changed back to _server/logs/_.
```

> The directory that the `neo4j-admin database import` command creates to store information about the import process, is moved from _server/data/imports_ to _server/logs/_.

Author: ```suggestion
The directory that the `neo4j-admin database import` command creates to store information about the import process, is changed back to _server/logs/_.
```

Neo4j 2026.03::

> The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_  has been changed to _dbname-yyyy-MM-dd.HH.mm.ss/import.log_, as the `neo4j-admin database import` command creates a new directory from version 2026.03. +

Author: ```suggestion
The import log name _neo4j-admin-import-yyyy-MM-dd.HH.mm.ss.log_  has been changed to _dbname-yyyy-MM-dd.HH.mm.ss/import.log_, as the `neo4j-admin database import` command creates a new directory. +
```

The path to the directory is changed from `server/logs` to `server/data/imports/`.

> 

Author: ```suggestion
+
```
